lib/graph.js

```diff
@@ -1,27 +1,59 @@
 const path = require('path');
 
-const link = require('./util/link');
+const _ = require('lodash');
+
 const deps = require('./util/deps');
+const link = require('./util/link');
+const pfs = require('./util/pfs');
+const promise = require('./util/promise');
 
-module.exports = function(dir) {
+module.exports = function(dir, deptraceOpts) {
```

Comment:
Yeah. Though I'm not much more of a fan of passing a deptrace instance. This method is seems like it prepares for the graph. All the graphing is the in deptrace but needs the info to start. Going to upload in a bit a series of fixups that makes most of graph into a prepare function that is used by an active (what is locally on the system) and target (max satisfying on the github).